Apps/Sandcastle/gallery/Cesium OSM Buildings Styling.html

```diff
@@ -0,0 +1,260 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Worldwide 3D buildings with rich metadata from OpenStreetMap."
+    />
+    <meta
+      name="cesium-sandcastle-labels"
+      content="Beginner, Showcases, ion Assets"
+    />
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script
+      type="text/javascript"
+      src="../../../Build/CesiumUnminified/Cesium.js"
+      nomodule
+    ></script>
+    <script type="module" src="../load-cesium-es6.js"></script>
+  </head>
+
+  <body
+    class="sandcastle-loading"
+    data-sandcastle-bucket="bucket-requirejs.html"
+  >
+    <style>
+      @import url(../templates/bucket.css);
+    </style>
+    <div id="cesiumContainer" class="fullSize"></div>
+    <div id="loadingOverlay">
+      <h1>Loading...</h1>
+    </div>
+    <div id="toolbar"></div>
+    <script id="cesium_sandcastle_script">
+      function startup(Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+        var viewer = new Cesium.Viewer("cesiumContainer", {
+          terrainProvider: Cesium.createWorldTerrain(),
+        });
+
+        // Create the OSM buildings
+        var buildings = Cesium.createOsmBuildings();
+        viewer.scene.primitives.add(buildings);
+
+        // Set the initial camera to look at Hong Kong
+        viewer.scene.camera.flyTo({
+          destination: Cesium.Cartesian3.fromDegrees(-122.3472, 47.598, 370),
+          orientation: {
+            heading: Cesium.Math.toRadians(10),
+            pitch: Cesium.Math.toRadians(-10),
+          },
+        });
+
+        // Styling functions
+        function colorByHeight() {
+          buildings.style = new Cesium.Cesium3DTileStyle({
+            color: {
+              conditions: [
+                [
+                  "${feature['cesium#estimatedHeight']} >= 100",
+                  "color('#581845')",
+                ],
+                [
+                  "${feature['cesium#estimatedHeight']} >= 50",
+                  "color('#900C3F')",
+                ],
+                [
+                  "${feature['cesium#estimatedHeight']} >= 25",
+                  "color('#C70039')",
+                ],
+                [
+                  "${feature['cesium#estimatedHeight']} >= 10",
+                  "color('#FF5733')",
+                ],
+                ["true", "color('#FFC300')"],
+              ],
+            },
+          });
+        }
```

Comment:
I'd remove color by height here since it's no different than the NYC styling example we have. I want this code example to focus on some of the more unique things about styling OSM buildings.